src/main/java/seedu/trippie/command/ListExpenditureCommand.java

```diff
@@ -1,4 +1,27 @@
 package seedu.trippie.command;
 
-public class ListExpenditureCommand {
+import seedu.trippie.ExpenditureList;
+import seedu.trippie.Product;
+import seedu.trippie.Ui;
+
+import java.util.ArrayList;
+
+public class ListExpenditureCommand extends Command {
+
+    public ListExpenditureCommand(ArrayList<Product> expenditureList) {
+        if (expenditureList.isEmpty()) {
+            Ui.printLine();
+            System.out.println("There is currently nothing in your expenditure list.");
```

Comment:
This needs to be called from Ui as well, once its implemented.

Reply:
I think it is okay to put it here since this comment is command-specific.

Reply:
I think it is okay to call it here since it is command-specific